src/service/AnnounceService.ts

```diff
@@ -358,18 +358,39 @@ export class AnnounceService {
                 } else {
                     const signerAccount = await resolveMainAccount();
                     if (transactions.length == 1) {
-                        await this.announceSimple(
-                            signerAccount,
-                            transactions[0],
-                            providedMaxFee,
-                            minFeeMultiplier,
-                            generationHash,
-                            currency,
-                            transactionService,
-                            listener,
-                            ready,
-                            nodeAccount.name,
-                        );
+                        if (transactions[0] instanceof MultisigAccountModificationTransaction) {
```

Comment:
would it be better to use the transactionType instead?

Reply:
why do you think so?

Reply:
I mean you can use instanceof, but the SDK has already got a transactionType enum. could that be neat using the type?